OmgvaPOS/src/UserManagement/Controller/UserController.cs

```diff
@@ -30,7 +30,7 @@ public class UserController(IAuthenticationRepository authenticationRepository,
         // [ProducesResponseType(StatusCodes.Status401Unauthorized)] 
         [ProducesResponseType(StatusCodes.Status409Conflict)]
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-        public IActionResult SignIn([FromBody]SignInRequest signInRequest)
+        public IActionResult SignIn([FromBody]SignInRequest signInRequest) //sita reiks padaryti, kad tik admin ar owneriui butu butu
```

Comment:
Can you prepend your comment with TODO so that it would be highlighted by IDEs and easier to search? Also translate to English instead of Lithuanian please

Reply:
Fine